homeassistant/components/qvr_pro/__init__.py

```diff
@@ -33,6 +40,9 @@
                 vol.Required(CONF_USERNAME): cv.string,
                 vol.Required(CONF_PASSWORD): cv.string,
                 vol.Optional(CONF_PORT, default=DEFAULT_PORT): cv.port,
+                vol.Optional(CONF_PROTOCOL, default=DEFAULT_PROTOCOL): vol.All(
```

Comment:
It looks like this change is in conflict with ADR10. Please review ADR7 and ADR10 before merging changes to existing config schemas and make a comment if it's borderline.

https://github.com/home-assistant/architecture/blob/master/adr/0010-integration-configuration.md

What do you think?

Reply:
I think we should revert this PR, as it conflicts with architectural decisions made.

Reply:
Opened up PR #39385 to revert this.